FitnessApp.Web/Controllers/NutrientCategoryController.cs

```diff
@@ -114,6 +122,7 @@ public async Task UpdateAsync([FromBody] NutrientCategoryDto nutrientCategoryDto
             _validator.Validate(nutrientCategoryDto, "UpdateNutrientCategory");
 
             await _nutrientCategoryService.UpdateAsync(nutrientCategoryDto);
+            _eventBus.Publish(new LogEvent(Statuses.Success, "Update", nutrientCategoryDto.GetType().Name.Replace("Dto", ""), JsonSerializer.Serialize(nutrientCategoryDto)));
```

Comment:
move JsonSerializer.Serialize to LogEvent

Reply:
for all

Reply:
It is excellent decision! Done.